MdePkg/Include/Guid/DebugImageInfoTable.h

```diff
@@ -35,8 +35,13 @@ typedef struct {
 
 typedef struct {
   ///
-  /// Indicates the type of image info structure. For PE32 EFI images,
-  /// this is set to EFI_DEBUG_IMAGE_INFO_TYPE_NORMAL.
+  /// Indicates the available image info in this structure.
+  /// A value of 2 indicates that PdbPath and DebugBase fields are present

```

Comment:
I would prefer a size field to be able to extend the struct on demand without breaking backward-compatibility. Maybe we should also explicitly declare it as unstable, so it can be cleaned up in the future if needed.